web/src/components/molecules/Content/ContentForm/index.tsx

```diff
@@ -0,0 +1,181 @@
+import styled from "@emotion/styled";
+import Upload from "antd/lib/upload/Upload";
```

Comment:
Don't use directly

Reply:
Fixed, I should not depend totally on autocomplete, didn't notice.